Justfile

```diff
@@ -9,15 +9,11 @@ check-das:
 test-das:
     {{nocgo}} go test ./...
 
-# Need to refactor this recipe to use ko
-# dev-build TAG: check-das test-das
-#     docker buildx build \
-#     --rm \
-#     --platform "linux/amd64" \
-#     -f ./Dockerfile \
-#     --provenance=false \
-#     --load \
-#     -t {{TAG}} \
-#     .
-
-#     echo "Built: {{TAG}}"
+# user still has to configure KO_DOCKER_REPO for the destination of ko build image
+# please refer https://ko.build/get-started/
+ko-build VERSION:
+    COMMIT_HASH=$(git rev-parse HEAD) VERSION={{VERSION}} \
+    ko build \
+    ./cmd/diskautoscaler \
+    --bare -t {{VERSION}}
+    echo "Built: {{VERSION}}"
```

Comment:
Reconfigure this so it only does local builds. No commit hash is needed. You can just make this a static string, same as version if you want. Idea is that Justfile won't be executed in the CI process, only locally. But, if run locally, it will be used to assist local builds only and not publish a release.

Reply:
I think it's fine to have commit hash ... I gave a string "build_locally" as a static string for version. LMK if its ok with you.